.changeset/fifty-maps-drive.md

```diff
@@ -0,0 +1,36 @@
+---
+'@backstage/plugin-scaffolder-react': major
```

Comment:
Don't think we need to say that this is a `major` change though, but we can update the wording in a the changeset a little. It's a bugfix as the behaviour was wrong before I'm thinking.

```suggestion
'@backstage/plugin-scaffolder-react': patch
```